clang/include/clang/Basic/LangOptions.def

```diff
@@ -423,6 +423,14 @@ LANGOPT(RelativeCXXABIVTables, 1, 0,
 
 LANGOPT(ArmSveVectorBits, 32, 0, "SVE vector size in bits")
 
+ENUM_LANGOPT(DefaultSubGroupSizeType, SubGroupSizeType, 2,
+             SubGroupSizeType::None,
+             "Strategy via which sub-group is assigned for SYCL kernel "
```

Comment:
We're not consistent on sub-group (used here) vs sub group (used in DiagnosticSemaKinds.td), and subgroup (in AttrDocs.td). We should pick one spelling and use it consistently. I'd recommend subgroup as it's a perfectly cromulent word.